components/content_settings/core/browser/brave_content_settings_pref_provider.h

```diff
@@ -75,6 +75,8 @@ class BravePrefProvider : public PrefProvider,
   std::map<bool /* is_incognito */, std::vector<Rule>> cookie_rules_;
   std::map<bool /* is_incognito */, std::vector<Rule>> brave_cookie_rules_;
 
+  bool initialized_;
```

Comment:
we need a comment, explaining why this is needed

Reply:
add here https://github.com/brave/brave-core/pull/6126/files#diff-9981585c3d9184fc54b96697c59d2028R144